lib/src/components/NxList/__tests__/NxListLinkItem.test.tsx

```diff
@@ -4,76 +4,122 @@
-    expect(contentEl.find('span').at(0)).toHaveText('Test Item 1 Text');
-    expect(contentEl.find('span').at(1)).toMatchSelector('span.nx-list__subtext');
-    expect(contentEl.find('span').at(0)).toHaveText('Test Item 1 Text');
+  it('sets aria-disabled="true" and does not pass href value on the anchor ' +
+  'iff disabled prop is set to true', function() {
+    const renderAndGetAnchor = (props?: Partial<NxListLinkItemProps>) => quickRender(props).getByRole('link');
+
+    expect(renderAndGetAnchor()).toHaveAttribute('href', 'www.sonatype.com');
+    expect(renderAndGetAnchor()).toHaveAttribute('aria-disabled', 'false');
+
+    expect(renderAndGetAnchor({ disabled: false })).toHaveAttribute('href', 'www.sonatype.com');
+    expect(renderAndGetAnchor({ disabled: false })).toHaveAttribute('aria-disabled', 'false');
+
+    expect(renderAndGetAnchor({ disabled: null })).toHaveAttribute('href', 'www.sonatype.com');
+    expect(renderAndGetAnchor({ disabled: null })).toHaveAttribute('aria-disabled', 'false');
+
+    expect(renderAndGetAnchor({ disabled: true })).not.toHaveAttribute('href');
+    expect(renderAndGetAnchor({ disabled: true })).toHaveAttribute('aria-disabled', 'true');
   });
 
-  it('sets aria-selected and aria-current to true iff selected is true', function() {
-    expect(getShallow()).toHaveProp('aria-selected', undefined);
-    expect(getShallow()).toHaveProp('aria-current', undefined);
+  it('sets aria-selected and aria-current="true" to the list ' +
+    'iff selected prop is set to true', function() {
+    const renderAndGetList = (props?: Partial<NxListLinkItemProps>) => quickRender(props).getByRole('listitem');
+
+    expect(renderAndGetList()).not.toHaveAttribute('aria-selected');
+    expect(renderAndGetList()).not.toHaveAttribute('aria-current');
 
-    expect(getShallow({ selected: true })).toHaveProp('aria-selected', true);
-    expect(getShallow({ selected: true })).toHaveProp('aria-current', true);
+    expect(renderAndGetList({ selected: false })).toHaveAttribute('aria-selected', 'false');
+    expect(renderAndGetList({ selected: false })).toHaveAttribute('aria-current', 'false');
 
-    expect(getShallow({ selected: false })).toHaveProp('aria-selected', false);
-    expect(getShallow({ selected: false })).toHaveProp('aria-current', false);
+    expect(renderAndGetList({ selected: null })).not.toHaveAttribute('aria-selected');
+    expect(renderAndGetList({ selected: null })).not.toHaveAttribute('aria-current');
+
+    expect(renderAndGetList({ selected: true })).toHaveAttribute('aria-selected', 'true');
+    expect(renderAndGetList({ selected: true })).toHaveAttribute('aria-current', 'true');
+  });
+
+  it('renders children correctly', function() {
+    const children = [
+      <NxList.Text data-testid="text" key="1">Test Item 1 Text</NxList.Text>,
+      <NxList.Subtext data-testid="subtext" key="2">Test Item 1 Subtext</NxList.Subtext>
```

Comment:
Pretty sure these should be indented two more levels

Reply:
To indent these two more levels, I would need to move the open bracket to the new line to avoid eslint complaints.
Updated here 9bf09bdb478fd4698d1c35aa2a57734639026e88.

Reply:
> To indent these two more levels, I would need to move the open bracket to the new line to avoid eslint complaints.
Updated here https://github.com/sonatype/sonatype-react-shared-components/commit/9bf09bdb478fd4698d1c35aa2a57734639026e88.

Huh, that's surprising.  It seems the react eslint plugin behaves differently from the base eslint indentation logic